nservicebus/upgrades/support-policy.md

```diff
@@ -9,7 +9,7 @@ related:
 
 ## Supported Versions
 
-Supported versions include any major version released within the last 3 years. Within a supported major version, the latest minor version is supported, or any minor version released within the last year.
+Supported versions include any major version released within the last 3 years. Within a supported major version, the latest minor version is supported. For the NServiceBus package only (i.e. excluding downstream packages such as [transports](/nservicebus/transports/) and [persistence](/nservicebus/persistence/)), any minor version released within the last year is supported.
```

Comment:
@adamralph - what you have is currently not clear. 

I suggest the following. I also discussed this with @kbaley.

NServiceBus:
Particular Software supports all major versions released within the last 3 years, including any minor versions released within that timeframe.  

For other downstream packages:
However, for downstream packages that use NServiceBus, for example, all the [transports](/nservicebus/transports/), [persistences](/nservicebus/persistence/), [containers](/nservicebus/containers/), etc.,  Particular Software will only support the latest minor version of any major versions that were released within the last three years.